package.json

```diff
@@ -19,7 +19,9 @@
     "@angular/platform-browser": "2.0.0-rc.1",
     "@angular/platform-browser-dynamic": "2.0.0-rc.1",
     "@angular/router": "2.0.0-rc.1",
+    "angular2": "^2.0.0-beta.17",
```

Comment:
@narainsagar we shouldnt be using rc1 and beta.17 at the same time, need to figure out how to use only RC1